src/index.css

```
@@ -456,3 +456,16 @@ a:hover {
   box-shadow: 0px 3px var(--accent-color);
   margin-bottom: 1em;
 }
+
+.paramContainer {
+  margin: 20px;
+  border: 1px solid gray;
+  padding: 8px;
+  background-color: var(--hdr-color);
+  border-radius: 5px;
+}
+
+.paramContainerText {
+  font-size: 15px;
```

please use relative units
```suggestion
  font-size: 1.5rem;
```